### DISCUSSION TITLE
Now trivy fs doesn't recognize binary file.

### DISCUSSION THREAD

**Discussion by rohitagg2020** [aquasecurity/trivy#1358] [Q&A] [answered]
Hi,

We were running trivy on our go binary file with versions prior to 1.20. It used to scan the binary file. But from 1.20 onwards, it stopped recognising the binary file. Also I see that from `trivy fs` command, `input` field is removed.  What is the alternative to that ?

## Prior to version 1.20:
Input:
trivy fs <BINARY_FILE_LOCATION>
Output:

INFO    Number of language-specific files: 1
INFO    Detecting gobinary vulnerabilities...
 
. (gobinary)
Total: 0 (HIGH: 0)
 

## Version 1.20 onwards:
Input:
trivy fs <BINARY_FILE_LOCATION>
Output:

INFO	Number of language-specific files: 0

**Comment by renuy** [comment]
+1

**Comment by knqyf263** [comment, marked as answer]
Duplicate of https://github.com/aquasecurity/trivy/issues/1359
